buildSrc/src/test/java/org/elasticsearch/test/VersionCollectionTests.java

```diff
@@ -0,0 +1,166 @@
+package org.elasticsearch.test;/*
```

Comment:
This package statement is in the wrong place (it should be after the copyright header).